fastdds_python/src/swig/fastdds.i

```diff
@@ -123,6 +123,7 @@ namespace builtin {
 %include "fastcdr/xcdr/optional.i"
 #endif
 
+%include "fastdds/LibrarySettings.i"
```

Comment:
I suggest adding a task to ensure to clean this repository up and leave only the interface `.i` files that are really exported into the python language binding. For instance, the `fastrtps/LibrarySettingsAttributes.i` file can be removed from the commented list and the corresponding file could be removed from the repository.

Reply:
I've taken a note on the 3.0 planning sheet (*Fast DDS Python - Cleanup not needed .i files*)